crates/op-rbuilder/src/builders/flashblocks/payload.rs

```diff
@@ -294,10 +294,20 @@ where
         let builder_txs = if ctx.attributes().no_tx_pool {
             vec![]
         } else {
-            self.builder_tx
-                .add_builder_txs(&state_provider, &mut info, &ctx, &mut state)?
+            match self
+                .builder_tx
+                .add_builder_txs(&state_provider, &mut info, &ctx, &mut state)
+            {
+                Ok(builder_txs) => builder_txs,
+                Err(e) => {
+                    error!(target: "payload_builder", "Error adding builder txs to fallback block: {}", e);
+                    vec![]
```

Comment:
is it okay to not include the builder txs? what are the consequences of not including them?

Reply:
but I think we can fallback to the regular builder tx if the flashblocks contract call fails so chain monitor can still mark the flashblock as something we built